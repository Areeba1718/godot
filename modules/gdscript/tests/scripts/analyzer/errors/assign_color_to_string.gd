# `Variant::can_convert_strict()` allows `String -> Color`, `but doesn't allow `Color -> String`.
# This test is to ensure that such asymmetric conversions are handled correctly for `Variant` types.

func test():
	var c: Color = Color.RED # Test non-constant.
	var s: String = c
