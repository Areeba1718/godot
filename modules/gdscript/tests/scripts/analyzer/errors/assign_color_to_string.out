GDTEST_ANALYZER_ERROR
Cannot assign a value of type Color to variable "s" with specified type String.
